### ISSUE TITLE
Accessibility: 

### ISSUE THREAD

**Issue by Divine-peace** [jupyterhub/outreachy#77]
### What page is this for?

https://jupyter.org/governance/intro.html

### WAVE accessibility report

![juper org](https://user-images.githubusercontent.com/103381382/196271636-fed3ff5a-bb9e-446c-a438-a39bced9135e.png)

**Suggestions for Form labels present but do not contain any content:**
Ensure that the form label contains text that describes the function of the associated form control. Labels are not required for image, submit, reset, button, or hidden form controls. If a label is not necessary visually, a descriptive title attribute may be added to the form control.
From the code evaluation, I suggest that a label element should not be used as a container for icons rather a div element.

**Suggestions for Multiple form labels:** 
Ensure that at most one label element is associated with the form control. If multiple form labels are necessary, use aria-labelledby. I would recommend simply using aria-labelledby instead of aria-label for multiple form labels.

**Suggestions for Language missing or invalid:**
I suggest that a lang attribute should be used instead of a style attribute to indicate the language of an html document.

**Suggestions for Empty buttons:**
Place text content within the <button> element or give the <input> element a value attribute.

**Suggestions for Redundant link Alert:**
If possible, combine the redundant links into one link and remove any redundant text or alternative text (for example, if a product image and product name are in the same link, the image can usually be given alt="").

**Suggestions for Features:**
Ensure that the alternative text presents the content of the image and/or the function of the link. If the full content and function of the link is presented in text within the link (an image and a text caption both within the same link, for example), then the image should generally be given empty/null alternative text (alt="") to avoid redundancy.
Ensure that  labels are accurate, descriptive, succinct, and that they are associated with the correct form control element.

**Suggestions for Stuctured Elements:**
Headings facilitate page navigation for users of assistive technologies. They also provide semantic and visual meaning and structure to the document. From the code evaluation, I suggest that the page should have only one H1 tag which helps convey the title of the page..

### The HTML element to be changed

```html
Errors for  (Empty form labels):

<label for="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-3">
<i class="fas fa-chevron-down"></i>
</label>

<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</label>

Empty form label
<label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</label>

Multiple form labels:

<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">

<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">

Language missing or invalid:
<html style="margin-left: 380px;">

Empty button:
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>


Redundant link Alert:
<a class="reference internal" href="overview.html">
<span class="doc std std-doc">
Overview of new (2022) Governance Model
</span>
</a>
</span>
</button>

3 Heading level 1:

<a class="navbar-brand text-wrap" href="index.html">
Heading level 1
<h1 class="site-logo" id="site-title">
Project Jupyter Governance
</h1>
</a>

<div id="jb-print-docs-body" class="onlyprint">
Heading level 1
<h1>
Project Jupyter Governance
</h1>

<section class="tex2jax_ignore mathjax_ignore" id="project-jupyter-governance">
Heading level 1
<h1>
Project Jupyter Governance
<a class="headerlink" href="#project-jupyter-governance" title="Permalink to this headline">
#
</a>
</h1>
```

### Your proposed HTML (or CSS), after change

```html
Errors for  (Empty form labels):

HTML:
<label for="toctree-checkbox-1" class=”visuallyhidden”>Select</label>
<i class="fas fa-chevron-down"></i>

CSS:
.visuallyhidden{
border: 0;
clip: rect (0 0 0 0);
height: 1px;
margin:-1px;
overflow: hidden;
padding: 0;
position: absolute;
width: 1px;
}

HTML:
<div class="headerbtn" data-toggle="tooltip" data-placement="right" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>


Errors for  (Multiple form labels):
HTML:
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-labelledby="Toggle navigation sidebar">

HTML:
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-labelledby="Toggle in-page Table of Contents">
</span>
</div>



HTML:
<div class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</div>

Error for  (Language missing or invalid) :
<html  lang=”en” >

Error for  (Empty button) :

<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="Toggle Fullscreen" data-original-title="Fullscreen mode" aria-label=”Toggle Fullscreen”>
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>


Alert for Redundant Link:

<h1 id=”overview”>
Jupyter Governance Overview
<a class="headerlink" href="#jupyter-governance-overview" title="Permalink to this headline">
#
</a>
</h1>

<a class="reference internal" href="overview.html#overview">
<span class="doc std std-doc">
Overview of new (2022) Governance Model
</span>
</a>


 3 (Heading Level 1):

<h2>
Project Jupyter Governance
<a class="headerlink" href="#project-jupyter-governance" title="Permalink to this headline">
#
</a>
</h2>



<div id="jb-print-docs-body" class="onlyprint">
<h2>
Project Jupyter Governance
</h2>

<div id="print-main-content">
<div id="jb-print-toc">
<div>
<h3>
Contents
</h3>
</div>
```

**Comment by minrk** [comment]
Thanks for the suggestions! The [aria-labelled-by](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) attribute should contain an _id_ of another element containing the label text. It should not be the text itself.

**Comment by minrk** [comment]
@Divine-peace please do!

**Comment by Divine-peace** [comment]
A big thanks to you sir

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
